Objects/Converters/ConverterRevit/ConverterRevitShared/ConversionUtils.cs

```diff
@@ -251,10 +251,11 @@ public void GetAllRevitParamsAndIds(Base speckleElement, DB.Element revitElement
 
       speckleElement["worksetId"] = revitElement.WorksetId.ToString();
 
+      // assign the category *if* it hasn't already been assigned (eg, DirectShape conversion should already have assigned the category).
       // for the category, we should be mirroring how we handle parsing of built-in-categories for revit 2023/2024
       // this is because different built-in-categories may have the same name, eg "OST_Railings" and "OST_StairsRailing" both have a Category name of "Railing"
       var category = revitElement.Category;
-      if (category is not null)
+      if (category is null)
       {
         var categoryName = revitElement.Category.Name;
```

Comment:
Is this right? first line in this statement wants `revitElement.Category.Name`, and `category = revitElement.Category` on the line above this one.

This means that getting the category name will throw an error as `revitElement.Category` is null in your changes

Reply:
Resolved 1 second after my comment